### ISSUE TITLE
No support for revprop on <tm> element in PDF output

### ISSUE THREAD

**Issue by easirois** [dita-ot/dita-ot#3214]
## Expected Behavior
The content in a topic has an `@rev` with an associated revprop value in a ditaval.  In this case, the background for the content has a particular color associated with it.  So I would expect the PDF to display the appropriate background color for the text in the PDF output.

## Actual Behavior
The appriate color should be displayed for content within `<tm>` when `@rev`  and revprop specified in a ditaval.

## Possible Solution
Add call to commonattributes prior to base apply-template reference.
```
<fo:inline xsl:use-attribute-sets="tm">
			<xsl:call-template name="commonattributes"/>
			<xsl:apply-templates/>
```

## Steps to Reproduce
Content:
`<tm rev="test-new" status="new" tmtype="reg">Test</tm>`

Ditaval:
`<revprop action="flag" val="test-new" backcolor="#A6F28D" changebar="style:solid;width:1pt;offset:5pt;color:#A6F28D"/>`

## Environment

* DITA-OT version: 2.3.1 to develop
* Operating system and version:
  Window 10 x8664
* How did you run DITA-OT?
oXygen transformation scenario 
* Transformation type:
PDF2

**Comment by easirois** [comment]
working on submitting a pull request for this issue.

**Comment by robander** [comment]
Fixed with #3215
